decode_test.go

```diff
@@ -3759,7 +3773,7 @@ func TestDecodeInvalidTagTime(t *testing.T) {
 			name:          "Tag 1 with negative integer overflow",
 			data:          hexDecode("c13bffffffffffffffff"),
 			decodeToTypes: []reflect.Type{typeIntf, typeTime},
-			wantErrorMsg:  "cbor: cannot unmarshal tag into Go value of type time.Time",
+			wantErrorMsg:  "cbor: cannot unmarshal negative integer into Go value of type time.Time (-18446744073709551616 overflows Go's int64)",
```

Comment:
My understanding is that overflow was causing parse to return `math.Big` in these cases, so the previous error came from the default case of the type switch in parseToTime.